dags/openshift_nightlies/config/install/hypershift/rosa-ovn.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
While I appreciate that you have accounted for both ROSA and OSD management clusters, it's just too much code and complexity for something which does not fit with the deployment model. We know management clusters will be OSD only, lets stick to that and avoid complexity. My 0.02.